pkg/cache/v3/simple.go

```diff
@@ -319,6 +325,9 @@ func (cache *snapshotCache) respondDeltaWatches(ctx context.Context, info *statu
 			// so we don't want to respond or remove any existing resource watches
 			if res != nil {
 				delete(info.deltaWatches, key.ID)
+				if indexed, typ, ok := getForcePushEdsResourcesNames(responseType, res); ok {
```

Comment:
to other reviewers - I had problem understanding where we detect the that there's been no state change - it is hidden by GH on line 324 in the comment

Reply:
another thing - why is it called `indexed` when it's the name of the resources? 🤔